scripts/mitochondria_m2_wdl/MitochondriaPipeline.wdl

```diff
@@ -275,7 +277,7 @@ task RevertSam {
   }
   runtime {
     disks: "local-disk " + disk_size + " HDD"
-    memory: machine_mem + " MB"
+    memory: mem + " MB"
```

Comment:
You can leave this as `machine_mem` so that you don't have to add the `Int mem = machine_mem` line. `machine_mem` is already overrideable. Or if you want to make it clearer to the user/more consistent with other tasks you can just rename `machine_mem` to `mem` without making an additional variable.